examples/kubernetes/cluster/main.tf

```diff
@@ -1,25 +1,23 @@
-data "vkcs_kubernetes_clustertemplate" "ct" {
-  version = "1.24"
-}
-
 resource "vkcs_kubernetes_cluster" "k8s-cluster" {
-  depends_on = [
-    vkcs_networking_router_interface.k8s,
-  ]
+  name                = "k8s-cluster"
+  cluster_template_id = data.vkcs_kubernetes_clustertemplate.k8s_124.id
+  master_flavor       = data.vkcs_compute_flavor.basic.id
+  master_count        = 1
 
   labels = {
-    cloud_monitoring = "true"
-    kube_log_level   = "2"
+    cloud_monitoring         = "true"
+    kube_log_level           = "2"
+    clean_volumes            = "true"
+    master_volume_size       = "100"
+    cluster_node_volume_type = "ceph-hdd"
```

Comment:
it is default value, lets change to ceph-ssd

Reply:
changed, working but i dont think its good to use ssd EVERY run because of ssd shorter lifecycles even for big clouds and datacenters.